### DISCUSSION TITLE
Has publishing this repo as a Galaxy collection been considered?

### DISCUSSION THREAD

**Discussion by eirik-talberg** [techno-tim/k3s-ansible#369] [Questions] [answered]
Hi!

I would like to integrate the playbooks and roles described here as a part of a larger repo with configurations, orchestration etc. This would be somewhat easier if it could be added as a dependency using Ansible Galaxy, instead of downloading/cloning the repo and running it as-is. Has this been considered? Is there a technical reason why not?

Hope to hear what people think of this idea. :)

**Comment by timothystewart6** [comment, marked as answer]
https://github.com/techno-tim/k3s-ansible/pull/388
